modules/tfm/tfm/boards/common/nrf_provisioning.c

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include "tfm_plat_provisioning.h"
+#include "tfm_plat_otp.h"
+#include "tfm_hal_platform.h"
+#include "tfm_attest_hal.h"
+#include "hw_unique_key.h"
+#include "nrfx_nvmc.h"
+#include <nrf_cc3xx_platform.h>
+#include <nrf_cc3xx_platform_identity_key.h>
+#include "nrf_provisioning.h"
+
+int tfm_plat_provisioning_is_required(void)
+{
+    enum tfm_security_lifecycle_t lcs = tfm_attest_hal_get_security_lifecycle();
+
+    return lcs == TFM_SLC_PSA_ROT_PROVISIONING;
+}
+
+enum tfm_plat_err_t tfm_plat_provisioning_perform(void)
+{
+    enum tfm_security_lifecycle_t lcs = tfm_attest_hal_get_security_lifecycle();
+
+    /*
+     * Provisioning in NRF defines has two steps, the first step is to execute
+     * the identity_key_generation sample. When this sample is executed it will
```

Comment:
Didn't we create a new sample with the name Provisioning image? Included in this PR?

Reply:
you are right, I will change this